terraform-docs.sh

```diff
@@ -1,16 +1,18 @@
 #!/bin/bash
 
 # https://stackoverflow.com/questions/59895/get-the-source-directory-of-a-bash-script-from-within-the-script-itself/246128#246128
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+AWK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DIR="$( cd "$2" >/dev/null 2>&1 && pwd )"
 
 command -v awk > /dev/null 2>&1 || ( echo "awk not available"; exit 1)
 command -v terraform > /dev/null 2>&1 || ( echo "terraform not available"; exit 1)
 command -v terraform-docs > /dev/null 2>&1 || ( echo "terraform-docs not available"; exit 1)
 
 if [[ "$(terraform version | head -1)" =~ 0\.12 ]]; then
     TMP_FILE="$(mktemp /tmp/terraform-docs-XXXXXXXXXX)"
+    cd "$DIR" || exit
```

Comment:
Test for existence before creating the TMP_FILE. Should this also exit with error and a messaage, like the `command` tests above?

Reply:
updated